frontend/src/components/contracts/ContractDetails.tsx

```diff
@@ -29,15 +29,17 @@ export default class extends React.Component<Props, State> {
       .getContractInfo(this.props.accountId)
       .then((contractInfo) => {
         if (contractInfo) {
-          this.setState({
-            codeHash: contractInfo.codeHash,
-            transactionHash: contractInfo.transactionHash,
-            timestamp: contractInfo.timestamp,
-            locked: contractInfo.accessKeys.every(
-              (key: any) =>
-                key["access_key"]["permission"]["FunctionCall"] !== undefined
-            ),
-          });
+          if (contractInfo !== undefined) {
```

Comment:
I think we can live with a single check here :smile:

Reply:
Sorry Frol, I do not understand what do you mean...... give me some hint about this

Reply:
![image](https://user-images.githubusercontent.com/304265/82713316-fb971f00-9c60-11ea-91e4-2fa13edd9552.png)